### PR TITLE
SOC-1906 reported indicator on posts

### PR DESCRIPTION
**Pull request by rafalkalinski**
the reported indicator on posts in the post list and on replies on the post details page

https://wikia-inc.atlassian.net/browse/SOC-1906

### CODE REVIEW

File: front/main/app/components/post-reply.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ export default Ember.Component.extend(
 	DiscussionMoreOptionsMixin,
 	{
 		classNames: ['post-reply'],
-		classNameBindings: ['isNew', 'isDeleted', 'isParentDeleted'],
+		classNameBindings: ['isNew', 'isDeleted', 'isReported', 'isParentDeleted'],
```
Comment (tomasznapieralski) [inline front/main/app/components/post-reply.js]: where did you use other bindings? `isNew` and `isParentDeleted`?

Comment (rafalkalinski) [inline front/main/app/components/post-reply.js]: It comes from as a template parameter.

**Comment by Bloodmoon24** [comment]
LGTM

**Comment by rafalkalinski** [comment]
moved to https://github.com/Wikia/mercury/pull/2009

**Pull request closed without merging**